monkeytype/typing.py

```diff
@@ -26,13 +27,33 @@
 
 
 DUMMY_TYPED_DICT_NAME = 'DUMMY_NAME'
+DUMMY_REQUIRED_TYPED_DICT_NAME = 'REQUIRED_TYPED_DICT_NAME'
+DUMMY_OPTIONAL_TYPED_DICT_NAME = 'OPTIONAL_TYPED_DICT_NAME'
+REQUIRED_FIELD_PREFIX = 'REQUIRED__'
+OPTIONAL_FIELD_PREFIX = 'OPTIONAL__'
```

Comment:
It seems these are only used in `remove_requiredness_prefix`, and that function isn't used anywhere? Just remove?